memcpy_audio.S

```diff
@@ -85,7 +85,18 @@
 	pkhtb r6,r8,r7,ASR #16
 
 	stmia r0!, {r3,r4,r5,r6}
-	pop	{r4-r8,r14}	
+	pop	{r4-r8,r14}
```

Comment:
actually now I'm wondering if that push and pop of r14 is even necessary for (AUDIO_BLOCK_SAMPLES == 8) or for (AUDIO_BLOCK_SAMPLES == 4), since that register is not used in those cases.